jerry-core/api/jerry-snapshot.c

```diff
@@ -34,7 +34,7 @@
  *
  * @return configuration flags
  */
-static inline uint32_t JERRY_ATTR_ALWAYS_INLINE
+static JERRY_ATTR_ALWAYS_INLINE uint32_t
```

Comment:
That's not exactly as suggested. I've suggested `static inline JERRY_ATTR_ALWAYS_INLINE uint32_t`.

Reply:
invalid